### PR TITLE
[CCXDEV-9225] Add CI/CD

### PR DESCRIPTION
**Pull request by juandspy**
Add `pr_check.sh` and `build_deploy.sh` to start building the quay image.

The CI/CD automation has been added at https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/47688.

### CODE REVIEW

File: build_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright 2021 Red Hat, Inc
```
Comment (tisnik) [inline build_deploy.sh:2]: 2022

**Review by tisnik — APPROVED** [review]
ok

**Review by joselsegura — APPROVED** [review]

File: pr_check.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+# --------------------------------------------
+APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="insights-content-template-renderer"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/insights-content-template-renderer"
+COMPONENTS="insights-content-template-renderer"  # space-separated list of components to laod
+COMPONENTS_W_RESOURCES="insights-content-template-renderer"  # component to keep
+CACHE_FROM_LATEST_IMAGE="true"
+
+export IQE_PLUGINS="ccx"
+export IQE_MARKER_EXPRESSION="smoke" # ccx_data_pipeline_smoke does not exits (at least yet) as marker in the plugin
```
Comment (matysek) [inline pr_check.sh:30]: Leave this empty.

File: pr_check.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+export IQE_FILTER_EXPRESSION=""
```
Comment (matysek) [inline pr_check.sh:31]: And here put just:
```
export IQE_MARKER_EXPRESSION=""
# Workaround: There are no specific integration tests. Check that the service loads and iqe plugin works.
export IQE_FILTER_EXPRESSION="test_plugin_accessible"
```

**Review by matysek — CHANGES_REQUESTED** [review]

**Pull request merged**